WalletWasabi.Fluent/ViewModels/Wallets/LoadingViewModel.cs

```diff
@@ -18,7 +18,7 @@ public partial class LoadingViewModel : ActivatableViewModel
 
 	[AutoNotify] private double _percent;
 	[AutoNotify] private string _statusText = " "; // Should not be empty as we have to preserve the space in the view.
-	[AutoNotify] private volatile bool _isLoading;
+	[AutoNotify(SetterModifier = AccessModifier.None)] private volatile bool _isLoading;
```

Comment:
`volatile` seems like a leftover from a previous design, we can just simply remove it.